### DISCUSSION TITLE
How can I get the topovolume id associated to a specific volume zone name?

### DISCUSSION THREAD

**Discussion by rmanno91** [ansys/pyprimemesh#771] [Q&A] [answered]
I have a certain geometry that I load into pyprime-meshing with 6 volume zones as shown in the picture

![image](https://github.com/ansys/pyprimemesh/assets/107696364/19c28391-7d35-41f7-a266-ba8cce656517)

Is there a way I can get the topo volume id belonging to a specific volume zone name?

I have also added here the geometry file I am loading:
[UD_RVE.zip](https://github.com/ansys/pyprimemesh/files/15151402/UD_RVE.zip)

**Comment by waltersma** [comment, marked as answer]
Hi @rmanno91, as of today the easiest way to do this is to import the cad with a new part per body so you have a single volume zone per topo volume if possible before merging as needed.  I will add an enhancement request for this.  Thanks
